src/player.h

```diff
@@ -1603,7 +1602,8 @@ class player : public Character
         std::set<int> follower_ids;
         //Record of player stats, for posterity only
         stats lifetime_stats;
-
+        // route for overmap-scale travelling
+        std::vector<tripoint> omt_path;
```

Comment:
Don't think there's any reason not to put this right into `Character`

Reply:
NPCs currently use omt_path for their overmap travelling too, it was originally in npc.h, but I moved it to player.h so both players and NPCs can use it.
Will that just work from Character too?